README.md

```diff
@@ -1,42 +1,48 @@
 # Mixed Messages | from Codecademy
+
 ## Table of Contents
+
+- [Mixed Messages | from Codecademy](#mixed-messages--from-codecademy)
+  - [Table of Contents](#table-of-contents)
+  - [Project Overview](#project-overview)
   - [Process](#process)
-  - [Built using:](#built-using)
+  - [Built using](#built-using)
   - [What we learned](#what-we-learned)
   - [Contributors](#contributors)
-  - [Acknowledgements](#acknowledgements)
-## Project Overview
-For this project, you will build a message generator program. Every time a user runs a program, they should get a new, randomized output. You’re welcome to take the project in a couple of different forms, like an astrology generator, inspirational message, or nonsensical jokes. To make your program truly random, the message that it outputs should be made up of at least three different pieces of data. Take what you know of JavaScript syntax so far to build the program and customize it to your liking.
 
-Project Objectives:
-* Build a message generator program using JavaScript
-* Use Git version control
-* Use command line
-* Develop locally on your computer
+## Project Overview
 
+For this project, we built a "hype up" message generator program. Every time a user runs a program, they should get a new, randomized output meant to inspire confidence and good feelings for their day. 
 
 Prerequisites:
-* JavaScript
-* Git and GitHub
-* Command line
 
-## Process
+- JavaScript
+- Git and GitHub
+- Command line
 
+## Process
+- Planning completed using a private Discord server
+- Idea/language to be used developed by Dow-mp
+- HTML/CSS written by sharkba1t
+- json developed and fetched via Javascript by jlema
```

Comment:
JSON fetching via Javascript developed by jlema

Reply:
Change made, commited and pushed.